src/util/ssa_expr.cpp

```diff
@@ -81,15 +89,16 @@ Function: ssa_exprt::build_identifier
 
 \*******************************************************************/
 
-irep_idt ssa_exprt::build_identifier(
+std::pair<irep_idt, irep_idt> ssa_exprt::build_identifier(
   const exprt &expr,
   const irep_idt &l0,
   const irep_idt &l1,
   const irep_idt &l2)
 {
   std::ostringstream oss;
+  std::ostringstream l1_object_oss;
 
-  build_ssa_identifier_rec(expr, l0, l1, l2, oss);
+  build_ssa_identifier_rec(expr, l0, l1, l2, oss, l1_object_oss);
 
-  return oss.str();
+  return std::make_pair<irep_idt, irep_idt>(oss.str(), l1_object_oss.str());
```

Comment:
cpplint says: "For C++11-compatibility, omit template arguments from make_pair OR use pair directly OR if appropriate, construct a pair directly"